ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -0,0 +1,148 @@
+                        label=""
+                        value="false">
+            </v-radio>
+            <v-radio class="infinite-radio pt-15 ma-0"
+                        :hide-details="false"
+                        label=""
+                        value="true">
+            </v-radio>
+        </v-radio-group>
+      </v-col>
+      <v-col cols="auto">
+        <v-text-field id="life-years-field"
+                        autocomplete="off"
+                        :error-messages="lifeYearsMessage || ''"
+                        filled
+                        :disabled="lifeYearsDisabled"
+                        :hint="lifeYearsHint"
+                        persistent-hint
+                        placeholder="Length (years)"
+                        v-model="lifeYearsEdit"/>
+        <div class="pt-5">Infinite ($500.00 non-refundable)</div>
+      </v-col>
+    </v-row>
+    <v-row no-gutters class='pt-10' v-if="showTrustIndenture">
+      <v-col cols="2" :class="$style[length-trust-label]">
+        Trust Indenture
+      </v-col>
+      <v-col cols="1">
+        <v-checkbox class="trust-checkbox pa-0 ma-0"
+                    :hide-details="false"
+                    :hint="trustIndentureHint"
+                    label=""
+                    v-model="trustIndenture">
+        </v-checkbox>
+      </v-col>
+      <v-col cols="auto">
+        Trust Indenture (Optional)
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useGetters, useActions } from 'vuex-composition-helpers'
+import { LengthTrustIF, FeeSummaryIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+
+export default defineComponent({
+  props: {
+    defaultTrustIndenture: {
+      type: Boolean,
+      default: false
+    },
+    defaultLifeInfinite: {
+      type: Boolean,
+      default: false
+    },
+    defaultLifeYears: {
+      type: String,
+      default: ''
+    },
+    defaultRegistrationType: {
+      type: String,
+      default: ''
+    }
+  },
+  setup (props, { emit }) {
+    const { setLengthTrust } = useActions<any>(['setLengthTrust'])
+    const { setFeeSummary } = useActions<any>(['setFeeSummary'])
+    const { getLengthTrust } = useGetters<any>(['getLengthTrust'])
+    const { getFeeSummary } = useGetters<any>(['getFeeSummary'])
+    const lengthTrust: LengthTrustIF = getLengthTrust.value
+    const feeSummary: FeeSummaryIF = getFeeSummary.value
+    const localState = reactive({
+      registrationType: props.defaultRegistrationType,
+      trustIndenture: lengthTrust.trustIndenture,
+      lifeYearsDisabled: false,
+      lifeInfinite: lengthTrust.lifeInfinite,
+      lifeYearsEdit: (lengthTrust.lifeYears > 0) ? lengthTrust.lifeYears.toString() : '',
+      lifeYearsMessage: '',
+      trustIndentureHint: '',
+      lifeYearsHint: 'Minimum 1 year, Maximum 25 years ($5.00 per year)',
```

Comment:
5.00, always wonder if that should be a constant somewhere for easy refactor :)

Reply:
Agreed, as per our discussion this morning, the fees will be refactored as a configurable property.